autoconfigure/pom.xml

```diff
@@ -33,6 +35,7 @@
 
   <modules>
     <module>collector-eventhub</module>
+    <module>storage-applicationinsights</module>
```

Comment:
Can collector be used in-process? So we do not need a sidecar container hosting it? Also it looks to me this is a regular setup. Like [here](https://docs.google.com/document/d/1estHBpOEEdU0ri3GkbxBbKqHZRvEC-SakY6XZruvRBM/edit):

>Production configuration
>- Numerous collectors zipkin collectors per data centre – no UI enabled
>- One zipkin UI server per data centre – firewalled and hidden behind an apache web server to provide authentication and auditing. Not accessible for span collector service